config.js.template

```diff
@@ -106,4 +106,9 @@ module.exports = {
   // You should consider what components you are enabling in the /health endpoint when setting this.
   // Check the /health endpoint documentation for more info.
   considerHealthcheckWarnAsUnhealthy: true,
-};
\ No newline at end of file
+
+  // Configure default history sync mode for the wallets
+  // Accepted values are: 'polling_http_api', 'xpub_stream_ws', 'manual_stream_ws'
+  // If no value is configured the default of 'polling_http_api' will be used
```

Comment:
By default, should we try to use the xpub stream and fallback to the pooling if it's not available?

Reply:
When introducing new features that work as an alternative to existing behavior I think keeping the default as the "current behavior" is better for people that constantly upgrade to the latest version since we keep their wallets working the same way since not using this config will not have any impact.